plugins/arElasticSearchPlugin/lib/model/arElasticSearchModelBase.class.php

```diff
@@ -83,27 +88,35 @@ public static function serializeI18ns($id, array $classes, $options = array())
       $i18ns = $options['merge'];
     }
 
-    // Tableize class name
-    foreach ($classes as &$class)
+    foreach ($classes as $class)
     {
-      $class = str_replace('Qubit', '', $class);
-      $class = sfInflector::tableize($class);
-      $class .= '_i18n';
+      // Tableize class name
+      $table = str_replace('Qubit', '', $class);
+      $table = sfInflector::tableize($table);
+      $table .= '_i18n';
```

Comment:
Alternately I think you could get the table name via:

```
$class_i18n = $class."I18n";
$table = $class_i18n::TABLE_NAME;
```

Reply:
Thanks @djjuhasz, addressed in 0652427.